src/styles/annotator/highlights.scss

```diff
@@ -1,102 +1,167 @@
-$color-highlight: rgba(255, 255, 60, 0.4);
-$color-highlight-second: rgba(206, 206, 60, 0.4);
-$color-highlight-focused: rgba(156, 230, 255, 0.5);
-
-// Hide content from sighted users but make it visible to screen readers.
+// This module configures and applies styles for annotation highlights in
+// annotatable documents.
 //
-// Resources:
-// - https://webaim.org/techniques/css/invisiblecontent/ (see "CSS clip")
-// - https://cloudfour.com/thinks/see-no-evil-hidden-content-and-accessibility/#showing-additional-content-for-screen-readers
-// - https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements Tailwind's `sr-only` class
-@mixin screen-reader-only {
-  // Take the content out of the normal layout flow.
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  white-space: nowrap;
-
-  // Visually hide the content and prevent it from interfering with mouse/touch
-  // text selection by clipping it to an empty box. Compared to moving content with
-  // `top`/`left` this is less likely to cause the browser to scroll to a
-  // different part of the page when the hidden text has screen-reader focus.
-  clip: rect(0 0 0 0);
-  overflow: hidden;
+// The highlight elements themselves (`.hypothesis-highlight`) and overall
+// highlight visibility toggling via `.hypothesis-highlights-always-on` are
+// managed in the `highlighter` module.
+//
+// Styling differentation for annotation "clusters" is activated by the presence
+// of the `.hypothesis-highlights-clustered` class. This class and the values of
+// applicable root-level CSS variables are managed by the `cluster-styles`
```

Comment:
Obviously the `cluster-styles` module doesn't exist yet. It will as of the next PR, but I could pull these comments if they bug you.